sdk/iOS/OoyalaSkinSDK/OOReactBridge.m

```diff
@@ -89,10 +91,38 @@ @implementation OOReactBridge
       [self handleOverlay:[parameters objectForKey:@"clickUrl"]];
     } else if ([buttonName isEqualToString:@"PIP"]) {
       [self handlePip];
+    } else if ([buttonName isEqualToString:stereoscopicButtonName]) {
+      [self handleStereoscopic];
     }
   });
 }
 
+RCT_EXPORT_METHOD(handleTouchMove:(NSDictionary *)params) {
+  LOG(@"Touch event handle - event move");
```

Comment:
Let's remove this log because it will add a lot and it will be hard to read the log in realtime.